### PR TITLE
Fixed BulkListeners in RT

### PR DESCRIPTION
**Pull request by holailusoria**

### CODE REVIEW

File: Backendless/RT/Data/DataSubscription.cs [hunk 1]
```diff
@@ -5,7 +5,7 @@ public class DataSubscription : RTSubscription
   {
     public DataSubscription( RTDataEvents rtDataEvent, String tableName, IRTCallback callback ) : base( SubscriptionNames.OBJECTS_CHANGES, callback )
     {
-      PutOption( "event", Enum.GetName( typeof( RTDataEvents ), rtDataEvent ) );
+      PutOption( "event", Enum.GetName( typeof( RTDataEvents ), rtDataEvent ).Replace( "_", "-" ) );
```
Comment (markpiller) [inline Backendless/RT/Data/DataSubscription.cs:8]: @holailusoria why is this needed? Seems odd

Comment (holailusoria) [reply Backendless/RT/Data/DataSubscription.cs:8]: @markpiller we get the name of the event handler of type Enum and convert it to a string. Requires a name of type "bulk-create", "bulk-update" etc. But we have "bulk_create", "bulk-update" respectively. This was the reason why BulkListeners did not work. This change fixes this bug.
